resteasy/src/main/java/com/cerner/beadledom/resteasy/GzipContentEncodingFilter.java

```diff
@@ -15,7 +15,7 @@
 @Provider
 @Priority(Priorities.HEADER_DECORATOR)
 public class GzipContentEncodingFilter extends ServerContentEncodingAnnotationFilter {
-  public GzipContentEncodingFilter() {
+  GzipContentEncodingFilter() {
```

Comment:
Why did you narrow the scope?

Reply:
Oops. I was just testing something. Reverted here https://github.com/cerner/beadledom/pull/179/commits/fb70f680b7583aaf63887cbeb4987261b9fcdc4b